src/blocks/woo-portal/search/assets/scripts/vue/Woo.vue

```diff
@@ -45,21 +45,24 @@
     <!-- informatieverzoek-->
     <div v-if="detail.URL_informatieverzoek">
       <h2>Informatieverzoek</h2>
-      <h3 class="woo-portal-detail__subtitle">URL informatieverzoek</h3>
-      <a
-          :href="detail.URL_informatieverzoek"
-          class="btn"
-          target="_blank"
-      >
-        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
-          <g>
-            <path
-                d="M4.83063 11.1696C4.95069 11.2898 5.11355 11.3573 5.28339 11.3573C5.45329 11.3573 5.61615 11.2898 5.73621 11.1696L11.1762 5.7296C11.338 5.56783 11.4012 5.33205 11.3419 5.11105C11.2827 4.89012 11.1101 4.71751 10.8892 4.65829C10.6682 4.59907 10.4324 4.66225 10.2706 4.82403L4.83063 10.264C4.71043 10.3841 4.64293 10.5469 4.64293 10.7168C4.64293 10.8867 4.71043 11.0495 4.83063 11.1696ZM7.65143 11.2L5.28023 13.584C4.61143 14.2528 3.42743 14.1552 2.64023 13.3696C1.85305 12.584 1.74742 11.3968 2.41622 10.728L4.80022 8.34884C4.96199 8.18707 5.02517 7.9513 4.96595 7.7303C4.90674 7.50931 4.73413 7.3367 4.51314 7.27748C4.29221 7.21826 4.05637 7.28144 3.89459 7.44322L1.51219 9.82242C0.344168 10.9904 0.440221 12.9872 1.72657 14.2736H1.72662C2.36626 14.9248 3.23795 15.2959 4.15062 15.3056C4.52562 15.3101 4.89786 15.2403 5.24578 15.1003C5.5937 14.9602 5.91047 14.7527 6.17781 14.4896L8.55701 12.1104C8.71961 11.9486 8.78357 11.7124 8.72482 11.4907C8.66602 11.2689 8.49342 11.0954 8.27201 11.0355C8.05059 10.9755 7.81398 11.0382 7.65138 11.2L7.65143 11.2ZM14.2738 1.72642C12.9938 0.446416 10.9906 0.344016 9.82263 1.51043L7.44343 3.88963C7.28083 4.0514 7.21687 4.28766 7.27562 4.50936C7.33442 4.73108 7.50702 4.90463 7.72843 4.96458C7.94985 5.02452 8.18646 4.96182 8.34906 4.80004L10.7283 2.41604C11.3971 1.74724 12.5811 1.84484 13.3683 2.63042C14.1554 3.41604 14.253 4.60322 13.5842 5.28002L11.2002 7.65122C11.0385 7.81299 10.9753 8.04876 11.0345 8.26976C11.0937 8.49075 11.2663 8.66336 11.4873 8.72258C11.7083 8.78179 11.9441 8.71862 12.1059 8.55684L14.4883 6.17764C15.6562 5.00962 15.5602 3.01284 14.2738 1.72644L14.2738 1.72642Z"
-                fill="white"/>
-          </g>
-        </svg>
-        Bekijk informatieverzoek
-      </a>
+      <span v-if="!detail.URL_informatieverzoek__is_upload">
```

Comment:
@rmpel wat is de reden dat je een span gebruikt? Normaliter zou dit een `<div>` of een `<template>` tag zijn

Reply:
Ik heb een wrapper nodig die de layout niet beinvloedt. In knockout zou ik dat met een <!-- ko:if ... --> doen, maar dat heeft vuejs niet, en een div geeft gelijk een block-element wat voor layout issues kan zorgen.
Ik kan uiteraard ook de v-if op alle elementen zetten, wat zou jij doen?